setup.py

```diff
@@ -28,7 +28,6 @@
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.2',
```

Comment:
Yet another 3.2 mention here ;)
The versions are also listed in `docs/requirements.rst`. Could you please change them too? Then we’re done, I will test it, update the changelog and release the new version.